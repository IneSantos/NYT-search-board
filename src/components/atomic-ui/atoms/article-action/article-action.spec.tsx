import React from "react";
import { render } from '@testing-library/react';
import ArticleAction from "./article-action";

describe("ArticleAction atom tests", () => {
  test(`ArticleAction component render`, () => {
    const { container } = render(
      <ArticleAction name="ARTICLE ACTION"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
