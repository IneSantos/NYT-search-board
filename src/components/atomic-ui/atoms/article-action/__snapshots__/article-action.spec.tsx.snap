// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleAction atom tests ArticleAction component render 1`] = `
<a
  class="article-action"
>
   
  ARTICLE ACTION
   
</a>
`;
